test/e2e/data/capi-operator/capv-variables.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vsphere-variables
+  namespace: default
```

Comment:
This secret will have to live in the same namespace as the CAPIProvider, and the namespace should be created with secret

Reply:
Done